helm/cluster-azure/templates/_helpers.tpl

```diff
@@ -332,6 +355,24 @@ DESCRIPTION:
   destinationPorts: "6443"
   source: "95.179.153.65"
   sourcePorts: "*"
+- name: "allow_apiserver_from_virtual_network"
+  description: "Allow K8s API Server"
+  direction: "Inbound"
+  priority: 154
+  protocol: "*"
+  destination: "*"
+  destinationPorts: "6443"
+  source: "VirtualNetwork"
+  sourcePorts: "*"
+- name: "allow_apiserver_from_azure_lb"
+  description: "Allow K8s API Server"
+  direction: "Inbound"
+  priority: 155
+  protocol: "*"
+  destination: "*"
+  destinationPorts: "6443"
+  source: "AzureLoadBalancer"
```

Comment:
When doing this, do we also get the `default rules` in the NSG ? 

mostly we should get 

* VirtualNetwork -> VirtualNetwork ALLOW ALL
* AzureLoadBalancer -> Any ALLOW ALL
* Deny All

If we do then we don't need this extra rule, if we don't then we need the very same rule to be defined since it is also used to check all the other load balancers ( like the ingress one for exmaple ) which will use random ports

![image](https://user-images.githubusercontent.com/2085772/229549870-4e758e66-9324-4c44-ac1c-5c5f25e6a74f.png)